applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -27,6 +27,73 @@
  * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration
  * Store.
  *
+ * <p>
+ *     Azure Configuration Store operations allowed by the client are adding, retrieving, updating, and deleting
+ *     ConfigurationSettings, additionally more complex operations of listing settings or revision of a setting based
+ *     on a filter are supported as well.
+ * </p>
+ *
+ * Construct the client
+ * <pre>
+ *     {@code
+ *     String connectionString = <connection_string>;
+ *     ConfigurationClientCredentials credentials = new ConfigurationClientCredentials(connectionString);
+ *     ConfigurationAsyncClient client = ConfigurationAsyncClient.builder()
+ *          .credentials(credentials)
+ *          .build();
+ *     }
+ * </pre>
```

Comment:
Putting `{@code` inside a pre block is the wrong way around, I believe. Generate the JavaDoc and check. Also, don't indent the code like this, start with no indentation and then indent as relevant.

Reply:
Reading up on it there is a purpose and valid use case for `@code` in a pre block but this wasn't the right one.